dds/DCPS/TypeSupportImpl.h

```diff
@@ -36,6 +38,41 @@ template <typename Message> struct DDSTraits;
 
 template <typename Message> struct MarshalTraits;
 
+class OpenDDS_Dcps_Export SerializedSizeBound {
+public:
+  SerializedSizeBound()
+  : bounded_(false)
+  , bound_(0)
+  {
+  }
+
+  SerializedSizeBound(size_t bound)
```

Comment:
```suggestion
  explicit SerializedSizeBound(size_t bound)
```

Reply:
I think this is one of the rare case where we would want an implicit conversion, since the class is basically an optional integer value. In addition to that and probably supporting that is the fact that it would be a bit of a pain to go through and add all constructors to the tests were they would be needed if it was explicit.